x-pack/test/api_integration_basic/apis/aiops/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('aiops basic license', function () {
+    this.tags(['aiops', 'walterra']);
```

Comment:
Similar to my other comment, `walterra` looks like a left-over debug tag here.

Reply:
Fixed in [52c1343](https://github.com/elastic/kibana/pull/135431/commits/52c134308784e818d5808e78c3dbd1c0964e2e0f).